service/controller/clusterapi/v29/resource/clusterazs/delete.go

```diff
@@ -0,0 +1,9 @@
+package clusterazs
+
+import (
+	"context"
+)
+
+func (r *Resource) EnsureDeleted(ctx context.Context, obj interface{}) error {
+	return nil
```

Comment:
Is there any need for AZs on delete event? I can't think of anything.

Reply:
Me neither.

Reply:
Easy to add later in case those are needed.